js/bundle/src/decoder.ts

```diff
@@ -0,0 +1,169 @@
+import * as CBOR from 'cbor';
+
+export class Bundle {
+    private version_: Uint8Array;
```

Comment:
Sorry I'm not familiar with TS.
Why are you using trailing "_"?

Reply:
This was to avoid conflict with the `version` getter function, but I changed it to a public field.